packages/teleport/src/Support/Support.tsx

```diff
@@ -102,29 +115,61 @@ export const Support = ({
           </Box>
         </Flex>
       </Card>
-      <Box
-        border="1px solid"
-        borderColor="primary.light"
-        mt={4}
-        mb={10}
-        borderRadius={3}
-        px={5}
-        py={4}
-      >
-        <Text as="h5" mb={4} fontWeight="bold" caps>
-          Cluster Information
-        </Text>
-        <ClusterData title="Cluster Name" data={clusterId} />
-        <ClusterData title="Teleport Version" data={authVersion} />
-        <ClusterData title="Public Address" data={publicURL} />
+      <DataContainer title="Cluster Information">
+        <DataItem title="Cluster Name" data={clusterId} />
+        <DataItem title="Teleport Version" data={authVersion} />
+        <DataItem title="Public Address" data={publicURL} />
         {tunnelPublicAddress && (
-          <ClusterData title="Public SSH Tunnel" data={tunnelPublicAddress} />
+          <DataItem title="Public SSH Tunnel" data={tunnelPublicAddress} />
         )}
-      </Box>
+      </DataContainer>
+
+      {isCloud && (
+        <DataContainer title="Scheduled Upgrades">
+          <DataItem
+            title="Window Start Time"
+            data={
+              <span>
+                {selectedUpgradeWindowStart}
+                <Edit onClick={showScheduleUpgrade} ml="2" />
```

Comment:
anything clickable should have `cursor: pointer`, and i believe they should also have like a hover state so when you hover over it it like changes color like most buttons do